packages/manager/src/features/CloudPulse/Alerts/AlertsResources/AlertsResources.tsx

```diff
@@ -401,6 +414,15 @@ export const AlertResources = React.memo((props: AlertResourcesProp) => {
             />
           </Grid>
         )}
+        <AlertsNoticeMessage text={errorText} variant="error" />
+        <AlertsNoticeMessage
+          text={
+            maxSelectionCount !== undefined
+              ? `You can select up to ${maxSelectionCount} resources.`
+              : undefined
+          }
+          variant="warning"
+        />
```

Comment:
hmm you would get an empty notice..

```suggestion
{maxSelectionCount !== undefined && (
        <AlertsNoticeMessage
          text=`You can select up to ${maxSelectionCount} resources.`
          variant="warning"
        />
      )
```

Reply:
@abailly-akamai , this is not needed as AlertsNoticeMessage component handles this, if the text is undefined it returns a empty component rather a empty notice

Reply:
Please clean up nonetheless

Reply:
@abailly-akamai , done this